src/index.js

```diff
@@ -40,19 +42,26 @@ function toMatchImageSnapshot(received, { customSnapshotIdentifier = '', customD
     customDiffConfig,
   });
   let pass = true;
-  if (result.updated) {
+  let message = '';
+  if (result.result === ResultTypes.UPDATE) {
```

Comment:
May want to rename the `result` variable in this file to make it more friendly to read. `result.result` is not the best. Maybe `comparison`? That way it is `comparison.result` and `comparison.diffOutputPath`